web/crux/src/app/image/image.mapper.ts

```diff
@@ -53,7 +53,8 @@ export default class ImageMapper {
       expose: config.expose,
       ingress: toPrismaJson(config.ingress),
       configContainer: toPrismaJson(config.configContainer),
-      user: config.user ? config.user : null,
+      // Set user to the given value, if not null or use 0 if specifically 0, otherwise set to default: -1
+      user: config.user || config.user === 0 ? (config.user === 0 ? 0 : config.user) : -1,
```

Comment:
should be: `config.user || config.user === 0 ? config.user : -1`

Reply:
Not the same.